dist/formats/placeholder/frontend/schema.json

```diff
@@ -40,6 +40,10 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "analytics_identifier": {
```

Comment:
I don't see why not - imagine you're working with Google Analytics data, it would be nice to be able to find the identifiers in the API responses. In the related Whitehall changes I'm making, I'm also exposing them in the Organisations/World Locations APIs.